components/sysutils/nmap/Makefile

```diff
@@ -54,29 +52,33 @@ COMPONENT_PREP_ACTION = (cd $(@D)/libdnet-stripped && autoconf)
 
 COMPONENT_PRE_CONFIGURE_ACTION = $(CP) -rp $(SOURCE_DIR)/. $(@D)
```

Comment:
Instead of this you could use `CLONEY_MODE = copy` to achieve the same, I think.

Reply:
it just works

Reply:
Yes, it works, but `CLONEY_MODE = copy` is far far better. :-)

Reply:
done